compiler/rustc_serialize/src/json.rs

```diff
@@ -1195,7 +1195,8 @@ impl Json {
     /// Attempts to get a nested Json Object for each key in `keys`.
     /// If any key is found not to exist, `find_path` will return `None`.
     /// Otherwise, it will return the Json value associated with the final key.
-    pub fn find_path<'a>(&'a self, keys: &[&str]) -> Option<&'a Json> {
+    #[cfg(test)]
+    fn find_path<'a>(&'a self, keys: &[&str]) -> Option<&'a Json> {
```

Comment:
Can you split `#[cfg(test)]` changes into a separate PR? I feel like those are *much easier* to justify.

Reply:
Opened https://github.com/rust-lang/rust/pull/83197.